app/models/agents/twitter_user_agent.rb

```diff
@@ -5,26 +5,20 @@ class TwitterUserAgent < Agent
     cannot_receive_events!
 
     description <<-MD
-      The Twitter User Agent follows the timeline of a specified Twitter user.
-
+      The Twitter User Agent either follows the timeline of a specific Twitter user or follow your own home timeline including both your tweets and tweets from people whom you are following.
       #{twitter_dependencies_missing if dependencies_missing?}
-
       To be able to use this Agent you need to authenticate with Twitter in the [Services](/services) section first.
-
-      You must also provide the `username` of the Twitter user to monitor.
-
+      For the first option, you must provide the `username` of the Twitter user to monitor.
```

Comment:
Maybe "To follow a Twitter user set `choose_home_time_line` to `true` and provide the `username`."?